### PR TITLE
Submit homework

### PR DESCRIPTION
**Pull request by toru828**

### CODE REVIEW

File: app/Http/Controllers/Api/UsersController.php [hunk 1]
```diff
@@ -14,19 +14,55 @@ class UsersController extends Controller
...
+            if(isset($requestData['name']) && $requestData['name']) {
+                $users = $users->where('name', 'LIKE', '%' . $request['name']. '%');
+            }
+            if(isset($requestData['email']) && $requestData['email']) {
+                $users = $users->where('email', 'LIKE', '%' . $request['email']. '%');
+            }
+            if(isset($requestData['created_at']) && $requestData['created_at']) {
+                $users = $users->where('created_at', 'LIKE', '%' . $request['created_at']. '%');
+            }
+            return response()->json($users);
```
Comment (oanhhathi) [inline app/Http/Controllers/Api/UsersController.php]: You have to use: $users= $users->paginate(25) to get the data

Comment (oanhhathi) [reply app/Http/Controllers/Api/UsersController.php]: return response()->json($users->paginate(25);

**Review by oanhhathi — COMMENTED** [review]
@toru828 Please check my comment.

**Review by lappham93 — COMMENTED** [review]
Its LGTM.
However, in actual project, we should implement structurally in backend: Controller -> Service (business) -> Repository -> Model

**Pull request merged**